Makefile

```diff
@@ -173,7 +173,6 @@ analyze: $(JS)
 ifeq ($(DEV), ashell)
 	@cat fragments/prj.mdef.dev >> prj.mdef
```

Comment:
Patches accepted. :)

I wanted to commit the part that I had figured out so far to make some progress...